### ISSUE TITLE
MiniCPM-V 2.6 memory leak occurred !!!

### ISSUE THREAD

**Issue by Liwx1014** [abetlen/llama-cpp-python#1886]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [ ] I am running the latest code. Development is very rapid so there are no tagged versions as of now.
- [ ✅] I carefully followed the [README.md](https://github.com/abetlen/llama-cpp-python/blob/main/README.md).
- [ ✅] I [searched using keywords relevant to my issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests) to make sure that I am creating a new issue that is not already open (or closed).
- [ ✅] I reviewed the [Discussions](https://github.com/abetlen/llama-cpp-python/discussions), and have a new bug or useful enhancement to share.

# Expected Behavior
MiniCPM-V 2.6 memory leak ?

# Current Behavior
I am testing [minicpm-v-2.6](https://huggingface.co/openbmb/MiniCPM-V-2_6-gguf) ,here is part of my test code:
![37a5d00121d9619b47c64c17b35131b](https://github.com/user-attachments/assets/2163bd7a-962f-41e4-8ec5-773fa6a7026d)

when i run code ,it adds 10MB per inference,i use memory_profiler find in  https://github.com/abetlen/llama-cpp-python/blob/2bc1d97c9672320828e70dc8293d5f8754682109/llama_cpp/llama_chat_format.py#L2839

![a6879c9732cf5cd576859e126b66faf](https://github.com/user-attachments/assets/070b8d5c-9471-4aa5-9f3d-4272c271ad06)

I thought at first that this variable "embed" wasn't being released,I added the code manual release on line 2856,but the program reported an error. 
then I replace "embed" with "self._last_image_embed",Memory leaks continue to occur.
# Environment and Context

Please provide detailed information about your computer setup. This is important in case the issue is not reproducible except for under certain specific conditions.then

Example environment info:
```

Python 3.10.10
ubuntu 18.04
cudatoolkit 12.1
llama-cpp-python 0.2.90

```

**Comment by woojh3690** [comment]
I have the same issue. It might be related to ggerganov/llama.cpp#9879.

**Comment by Liwx1014** [comment]
> I have the same issue. It might be related to [ggerganov/llama.cpp#9879](https://github.com/ggerganov/llama.cpp/pull/9879).

ok,thanks!. By the way  Have you tried this solution? Whether this problem can be solved

**Comment by woojh3690** [comment]
No, I haven't tried applying the solution from ggerganov/llama.cpp#9879.

**Comment by woojh3690** [comment]
@Liwx1014 May be already fixed this issue. Can you try latest version? ggerganov/llama.cpp#11513

**Comment by Liwx1014** [comment]
> [@Liwx1014](https://github.com/Liwx1014) May be already fixed this issue. Can you try latest version? [ggml-org/llama.cpp#11513](https://github.com/ggml-org/llama.cpp/pull/11513)

I try this solution ,the bug has been solved.
